### ISSUE TITLE
Obsolete GO:0100035 negative regulation of transmembrane transport by transcription from RNA polymerase II promoter

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#13685]
Hello, 

I'd like to obsolete 
GO:0100035 negative regulation of transmembrane transport by transcription from RNA polymerase II promoter
and its child
GO:1900417 	negative regulation of transmembrane transport by negative regulation of transcription from RNA polymerase II promoter

Created 2 years ago, no annotations yet, and no reference in the definition, so the potential usage is not clear. 

@cmungall you created this, also with obol as a reference. Let me know if this term is needed for anything. 

Thanks, Pascale

**Comment by pgaudet** [comment]
Obsoletion notice sent. 
Dear all, 

The  proposal has been made to obsolete: "GO:0100035 negative regulation of transmembrane transport by transcription from RNA polymerase II promoter”. The reason for obsoletion is that while the term was created 2 years ago, no annotations yet, and no reference in the definition, so the potential usage is not clear. 

There are no annotations and the term is not part of any slim. 

We are opening a comment period for this proposed obsoletion. We'd like to proceed and obsolete this term on July 5, 2017

*** Unless objections are received by July 5, 2017 , we will assume that you agree to this change. ***

Thanks,

Pascale

**Comment by cmungall** [comment]
huh, so I did. There's a big block of mine at GO:01000*, all added en mass, same timestamp. Someone must have told me to do it! I won't miss these.

**Comment by cmungall** [comment]
How are you determining there are no annotations?

http://amigo.geneontology.org/amigo/term/GO:1900417

There is a pombase annotation there, cc @ValWood

**Comment by cmungall** [comment]
Why would I go about making a term as odd as GO:0100035? OK, it's a bit involved.

But basically we had GO:1900417, and it had annotations. The problem is GO:1900417 had no (computable) def and thus no way to map to a GOCAM, reason with it etc.

We originally used a DP with 'results_in' relation... but this was odd, 

If we go back to
http://wiki.geneontology.org/index.php/Ontology_meeting_2015-04-02

and the email thread "results in outliers"

I said

```
Previously we had agreed on:

id: GO:2000531
name: regulation of fatty acid biosynthetic process by regulation of transcription from RNA polymerase II promoter
intersection_of: GO:0006357 ! regulation of transcription from RNA polymerase II promoter
intersection_of: results_in GO:0042304 ! regulation of fatty acid biosynthetic process

==>

id: GO:2000531
name: regulation of fatty acid biosynthetic process by regulation of transcription from RNA polymerase II promoter
intersection_of: GO:0050789 ! regulation of biological process
intersection_of: regulates 'NEW:transcription from RNA polymerase II promoter' and regulates some 'fatty acid biosynthetic process'

I can automate this. There is a fairly mild inflation of terms here (but we need to name the inner expression so long as we are wed to obo)

However, some help looking at the outliers below would be useful. This is the email I sent after the last discussion:
```

so there was a  mild inflation of regulation terms, in order to provide axiom support for the pre-existing terms

TL;DR - the preexisting terms should maybe have never been in there in the first place, but they were, we can get rid of a lot more when we switch to GOCAM but for now if there are annotations we have to be careful

**Comment by RLovering** [comment]
Hi Pascale 
It is a big concern that we are being told that there are no annotations to a term, when actually there are. This is not the first time this has happened. If we are told there are no annotations then people will not be checking the term is important to them. Please can you work out what is going on that you don't see annotations that exist. Previous GO editors didn't have this problem so maybe you need to discuss what approach they have taken and follow this instead of how you are doing this.

I really don't have time to have to check every term being obsoleted which is listed as having no annotations. Certainly it would also add to all biocurators work if this was an additional check that we all had to do individually.

Many thanks

Ruth

**Comment by pgaudet** [comment]
Hi @RLovering 

Apologies. There was indeed one annotation but to a child, 'negative regulation of transmembrane transport by negative regulation of transcription from RNA polymerase II promoter'. For some reason that didn't show up in QuickGO; I expected it would. 
@tonysawfordebi do you know why this is not shown ? Are the 'regulates' relations not used for displaying annotations ? 

Thanks, Pascale

**Comment by pgaudet** [comment]
Hi @cmungall @kltm 
In AmiGO I dont understand why when I am on this page: 
http://amigo.geneontology.org/amigo/search/annotation?q=GO:0100035

I see under GO Class direct all the following: 
GO class (direct)
(215)	nuclear-transcribed mRNA catabolic process, no-go decay
(212)	dopamine neurotransmitter receptor activity, coupled via Gi/Go
(9)	negative regulation of transmembrane transport by negative regulation of transcription from RNA polymerase II promoter
(2)	courtship behavior
(2)	plasma membrane
(2)	potassium ion transport
(2)	prefoldin complex
(2)	voltage-gated cation channel activity
(2)	voltage-gated potassium channel activity
(2)	voltage-gated potassium channel complex
(1)	G-protein coupled receptor activity
(1)	G-protein coupled receptor signaling pathway
(1)	behavioral response to ether
(1)	beta-tubulin binding
(1)	centrosome cycle
(1)	cytoplasm
(1)	establishment or maintenance of neuroblast polarity
(1)	flight behavior
(1)	integral component of membrane
(1)	learning
(1)	learning or memory
(1)	microtubule-based process
(1)	mitotic spindle assembly
(1)	perineurial glial growth

--- 
Hence I have been trying to gt the information from QuickGO. 

Thank you in advance for your help ! 

Pascale

**Comment by RLovering** [comment]
Hi Pascale

The default for the QuickGO has always been to only provide annotations to a smaller range of child terms than AmiGO. I'm sure I have mentioned this several times in GOC meetings pointing out that it may be confusing to our users to have these differences. However if you go to the filter option and select the GO identifier tab you will see listed: 
Default behaviour (is_a, part_of and occurs_in relationships only)
I would suggest you use the option:
is_a, part_of, occurs_in, regulates, positively_regulates and negatively_regulates
I think if you use all relations you will get the has_part child terms too.

Hope this helps

Ruth

**Comment by pgaudet** [comment]
Ah ! yes thanks ! I'll do this from now on. 

@ValWood is this term needed ?

**Comment by ValWood** [comment]
On 29/06/2017 10:40, pgaudet wrote:
>
> In AmiGO I dont understand why when I am on this page:
> http://amigo.geneontology.org/amigo/search/annotation?q=GO:0100035
>
> I see under GO Class direct all the following:
> GO class (direct)
> (215) nuclear-transcribed mRNA catabolic process, no-go decay
> (212) dopamine neurotransmitter receptor activity, coupled via Gi/Go
>


Pascale, Thank you,
I have moaned about this many times but I could not get anyone  else to 
agree with me!
If you are on a GO term, this link shows you ALL of the  direct 
annotations  for any association for any gene product gene product 
annotated to the gene in question. I think it should be confined to the 
annotations only in branches related to the GO term (descendants). This 
practice is, IMHO not intuitive, or useful. If you think it's strange, 
and I think it's strange its probably confusing the hell out of the users.


Re:

In AmiGO I don't understand why when I am on this page:
http://amigo.geneontology.org/amigo/search/annotation?q=GO:0100035 I see 
under GO Class direct all the following:
GO class (direct)
(215) nuclear-transcribed mRNA catabolic process, no-go decay
(212) dopamine neurotransmitter receptor activity, coupled via Gi/Go

**Comment by pgaudet** [comment]
Hi @ValWood 
I agree, that's not intuitive at all !! 

Thanks for the pointer. I go now obsolete away ;)

**Comment by RLovering** [comment]
Actually I think it is very weird that when we all agree that an entity regulating a process does not mean that entity is part of the process in AmiGO the regulating and process annotations are all lumped together as the default.

Not all functional analysis tools lump regulation and process annotations together, so if you are using these it is worth checking whether the tool is grouping regulation and process annotations together or keeping them as separate groups. If you are using a tool that keeps them separate then using the QuickGO default makes sense.

**Comment by ValWood** [comment]
Re
> 
Just to close this issue: https://github.com/geneontology/go-ontology/issues/13685#issuecomment-311988220
do you care about this annotation: 
GO59763
oca2
GO:1900417                                


nope, we can rehouse.

**Comment by ValWood** [comment]
I have wondered why QuickGO does not include regulates (it used to), and it seems sensible.
Especially as so much regulation is within process regulation...

**Comment by ValWood** [comment]
> Actually I think it is very weird that when we all agree that an entity regulating a process does not mean that entity is part of the process in AmiGO the regulating and process annotations are all lumped together as the default.

This seems to be a sensible default ...
For many process you would lose a lot of "within process" regulation annotation, where the regulation *is* part_of the process.
There is no reason why the default should only be part_of
If you went to "cell cycle" or "mitotic cell cycle" for fission yeast you could see no annotations if you exclude "regulates" relationship.
That would be misleading....

**Comment by cmungall** [comment]
There is a lot going on in this ticket.

Please file separate tickets in the amigo tracker for any amigo issues.

I'm going to undo this obsoletion, unfortunately it breaks things because as I mentioned above, there are terms that depend on the to be obsoleted term.

My fault, there is no sparql check for this yet, so your PR looked OK in travis

**Comment by cmungall** [comment]
OK, I undid #13752 with #13794

#13793 adds a SPARQL check that ensures the ontology is never in a state whereby live classes reference obsolete classes. We will discuss this and cascading delete policy on friday
